vulkano/src/swapchain/capabilities.rs

```diff
@@ -103,7 +111,9 @@ pub fn supported_present_modes_from_list<I>(elem: I) -> SupportedPresentModes
             vk::PRESENT_MODE_MAILBOX_KHR => result.mailbox = true,
             vk::PRESENT_MODE_FIFO_KHR => result.fifo = true,
             vk::PRESENT_MODE_FIFO_RELAXED_KHR => result.relaxed = true,
-            _ => panic!("Wrong value for vk::PresentModeKHR"),
+            vk::PRESENT_MODE_SHARED_DEMAND_REFRESH_KHR => result.shared_demand = true,
+            vk::PRESENT_MODE_SHARED_CONTINUOUS_REFRESH_KHR => result.shared_continuous = true,
+            _ => panic!("Wrong value for vk::PresentModeKHR: {}", e),
```

Comment:
Panicking here breaks existing applications if new present modes are added to Vulkan. Perhaps this should be changed into a `debug_assert!` instead?

```suggestion
            _ => debug_assert!(false, "Wrong value for vk::PresentModeKHR: {}", e),
```